mitodl/mitx-theme
# feat: update logo

#### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/3340

#### What's this PR do?
Updates the logo.

#### How should this be manually tested?

- Enable theming and use mitx-theme by following the [Readme](https://github.com/mitodl/mitx-theme)
- Add the following settings in the private.py
  - LMS_BASE_URL = "http://localhost:18000"
  - LOGO_URL_PNG_FOR_EMAIL = f"{LMS_BASE_URL}/static/mitx-theme/images/logo.png"
  - LOGO_URL = f"{LMS_BASE_URL}/static/mitx-theme/images/logo.svg"
  - LOGO_TRADEMARK_URL = f"{LMS_BASE_URL}/static/mitx-theme/images/mit-ol-logo.svg"
  - BULK_EMAIL_SEND_USING_EDX_ACE = True
- Verify the updated logos in the legacy theme and the MFEs
  - For any MFE, you will need to set the LOGO_URL and LOGO_TRADEMARK_URL in the .env.development
- Verify the logo in the Ace Emails

#### Screenshots (if appropriate)

<img width="1739" alt="Screenshot 2024-03-21 at 9 44 02 PM" src="https://github.com/mitodl/mitx-theme/assets/52656433/ab5430ed-c83d-42da-8fad-d76768ed96c9">
<img width="1739" alt="Screenshot 2024-03-21 at 9 44 32 PM" src="https://github.com/mitodl/mitx-theme/assets/52656433/e6418d99-719e-4ed2-a096-a2f3613fd508">
